code/src/components/NewThought.js

```diff
@@ -0,0 +1,53 @@
+
+import React, { useState } from 'react';
+
+export const NewThought = () => {
+  const [newThought, setNewThought] = useState('');
+
+  const HandleFormSubmit = (event) => {
+    event.preventDefault();
+    if (newThought.length < 5) {
+      return alert('Uh oh! Expand your thoughts to more than 5 letters!')
+    } else if (newThought.length > 140) {
+      return alert('Easy Proust, max length is 140 characters!')
+    } else {
+      const Submit = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({ message: newThought })
+      };
+      fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts', Submit)
```

Comment:
One thing I forgot was that this API could have been in an const. Maybe you wanted it this way tho.